ProGetAutomation/Functions/Get-ProGetAsset.ps1

```diff
@@ -0,0 +1,49 @@
+function Get-ProGetAsset
+{
+    <#
+        .SYNOPSIS
+        Used to get assets to the ProGet asset manager. 
+
+        .DESCRIPTION
+        This function gets assets to the proget asset manager. A session and assetDirectory is required. 

```

Comment:
Change `gets assets to the proget asset manager` to `gets assets from ProGet`.

Reply:
Done also changed this in Add-ProGetAsset